bin/test

```diff
@@ -6,5 +6,6 @@ chart="$1"
 
 cd "$chart"
 
+helm lint .
 [ -d test/unit ] && bats test/unit
-[ -d test/acceptance ] && bats test/acceptance
+# [ -d test/acceptance ] && bats test/acceptance
```

Comment:
Did you want this commented out?

Reply:
yeah, they're not really standardized and currently it's not working correctly because of a socket name length issue on docker-for-desktop. Not quite ready to include these tests as a part of the `bin/package` script (which automates packaging and tagging of the chart).